bundles/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/bus/HomematicBinding.java

```diff
@@ -183,15 +183,24 @@ protected void internalReceiveUpdate(String itemName, State newState) {
 
 	/**
 	 * Restarts the Homematic communicator if no messages arrive within a
-	 * configured time.
+	 * configured time or the reconnect interval is reached.
 	 */
 	@Override
 	protected void execute() {
-		long timeSinceLastEvent = (System.currentTimeMillis() - communicator.getLastEventTime()) / 1000;
-		if (timeSinceLastEvent > context.getConfig().getAliveInterval()) {
-			logger.info("No event since {} seconds, refreshing Homematic server connections", timeSinceLastEvent);
-			communicator.stop();
-			communicator.start();
+		if (context.getConfig().getReconnectInterval() == null) {
```

Comment:
I would prefer to remove this if-else block and rather initialise reconnectInterval with a reasonable value. What do you think?